articles/closing_the_trust_gap/part2.md

```diff
@@ -159,7 +159,7 @@ all in the metadata file. It can even be
 done in a way such that you do not
 have to specify the contract address!
 
-It gets better: In part 3 I will show you a system
+It gets better: In [part 3](./part3) I will show you a system
```

Comment:
Can you remove this, too?

Reply:
Sure, I just thought that it would be easy to read the whole sequence if there was a link to the final part on part 2.